docker/Dockerfile.armv7-unknown-linux-gnueabihf

```diff
@@ -1,28 +1,31 @@
+# This was built from ./Dockerfile.context
+FROM alsuren/cross-context:latest as context
+
 FROM ubuntu:16.04
```

Comment:
This idea turns out to be really powerful when creating custom images, because it allows you to provide `context` as an empty directory and `dockerfile` as something that uses this pattern to fetch build scripts etc.

In https://github.com/lcsfelix/reading-xiaomi-temp/pull/1/files I was previously using `context = "../cross/docker"` which is probably not something you want to check into your git repo, because it requires all developers to keep a copy of the `cross` repo in a specific location. 

TODO: 
- [ ] upload this `cross-context` docker image somewhere more official-sounding, and make sure that it gets uploaded and tagged with each release.
- [ ] revert the changes to this file, and produce a dedicated "example custom build image" dockerfile that uses this pattern.